src/main/java/org/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -63,7 +110,21 @@ private char detectDelimiter(Path file) throws IOException {
         }
     }
 
+    private static String removeHyphens(String string) {
+        if (string.startsWith("-")) {
+            string = string.substring(1);
+        }
+        if (string.endsWith("-")) {
+            string = string.substring(0, string.length() - 1);
+        }
```

Comment:
Just use 

    string = org.jabref.model.strings.StringUtil.removeStringAtTheEnd(string, "-");

(replace by normal import - i just want tos show you the package)